### ISSUE TITLE
UnexpectedNativeTypeException: expected map, got a string  in 1.7.0version

### ISSUE THREAD

**Issue by panda8z** [wmcmahan/react-native-calendar-events#225]
when I upgrade my `react-native-calendar-events: 1.6.4` to `1.7.0` a runtime crash occur in my Android App.
![image](https://user-images.githubusercontent.com/24646937/54105306-95ccc780-440d-11e9-8703-1666287fd05a.png)

![image](https://user-images.githubusercontent.com/24646937/54105274-851c5180-440d-11e9-905e-a19a8efad446.png)

### Environment

before:`react-native-calendar-events: 1.6.4`

after:`react-native-calendar-events: 1.7.0`
### Steps to Reproduce
use  `react-native-calendar-events:1.7.0` and code in my ReactNative below:

```javascript
 RNCalendarEvents.fetchAllEvents(this.state.startDate, this.state.endDate, ['1', '2']).then(events => {
            let arr = []
            if (events.length) {
                let id = 0;
                // console.log(JSON.stringify(events))
                let reminds = events.map(item => {
                    if (Platform.OS == 'ios') {
                        // 有提醒时间的逻辑
                        if (item.alarms && item.alarms.length && item.alarms[0].date) {
                            // item.startDate = item.startDate; 有提醒时间的逻辑
                        }
                    } else {
                        if (item.alarms && item.alarms.length && item.alarms[0].date) {
                            // let alarmTime = item.alarms[0].date;
                            // let startTime = item.startDate;
                            // let timeStamp = new Date(startTime).getTime() - (new Date(alarmTime).getTime() - new Date(startTime).getTime());
                            // item.startDate = new Date(timeStamp);
                        }
                    }
                    arr.push(id)
                    ++id;
                    return {
                        id: id,
                        name: item.title,
                        all_day: item.allDay ? '1' : '0',
                        // allDay: item.allDay,
                        // alarms: item.alarms ? item.alarms : false,
                        // startDate: item.startDate,
                        todo_start: new Date(item.startDate).getTime(),
                        todo_end: new Date(item.endDate).getTime(),
                        // endDate: item.endDate,
                        type: item.allDay ? '0' : '1',
                        todo_remind_time: item.alarms.length > 0 ? item.alarms[0].date ? new Date(item.alarms[0].date).getTime() : new Date(item.startDate).getTime() : '0',
                        start_at: new Date(item.startDate).getTime(),
                        // isRemind: item.alarms && item.alarms.length,
                        title: item.title
                    }
                });
```

**Comment by Octaner** [comment]
Similar problem:
<img width="803" alt="Screenshot 2019-03-11 at 18 53 33" src="https://user-images.githubusercontent.com/2323024/54137715-2ff13780-442f-11e9-985b-9126d500dbb4.png">

**Comment by hisothreed** [comment]
update to the latest version npm i --save react-native-calendar-events@latest

**Comment by wmcmahan** [comment]
Is fixed in `1.7.1`
